website/docs/r/custom_db_role.html.markdown

```diff
@@ -0,0 +1,164 @@
+
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find group_id in the official documentation.
+
+## Example Usage 
+
+```hcl
+resource "mongodbatlas_custom_db_role" "test_role" {
+  project_id = "<PROJECT-ID>"
+  role_name  = "myCustomRole"
+
+  actions {
+    action = "UPDATE"
+    resources {
+      collection_name = ""
+      database_name   = "anyDatabase"
+    }
+  }
+  actions {
+    action = "INSERT"
+    resources {
+      collection_name = ""
+      database_name   = "anyDatabase"
+    }
+  }
+  actions {
+    action = "REMOVE"
+    resources {
+      collection_name = ""
+      database_name   = "anyDatabase"
+    }
+  }
+}
+```
+
+## Example Usage with inherited roles
+
+```hcl
+resource "mongodbatlas_custom_db_role" "inherited_role_one" {
+  project_id = "<PROJECT-ID>"
+  role_name  = "insertRole"
+
+  actions {
+    action = "INSERT"
+    resources {
+      collection_name = ""
+      database_name   = "anyDatabase"
+    }
+  }
+}
+
+resource "mongodbatlas_custom_db_role" "inherited_role_two" {
+  project_id = "${mongodbatlas_custom_db_role.inherited_role_one.project_id}"
+  role_name  = "statusServerRole"
+
+  actions {
+    action = "SERVER_STATUS"
+    resources {
+      cluster = true
+    }
+  }
+}
+
+resource "mongodbatlas_custom_db_role" "test_role" {
+  project_id = "${mongodbatlas_custom_db_role.inherited_role_one.project_id}"
+  role_name  = "myCustomRole"
+
+  actions {
+    action = "UPDATE"
+    resources {
+      collection_name = ""
+      database_name   = "anyDatabase"
+    }
+  }
+  actions {
+    action = "REMOVE"
+    resources {
+      collection_name = ""
+      database_name   = "anyDatabase"
+    }
+  }
+
+  inherited_roles {
+    role_name     = "${mongodbatlas_custom_db_role.inherited_role_one.role_name}"
+    database_name = "admin"
+  }
+
+  inherited_roles {
+    role_name     = "${mongodbatlas_custom_db_role.inherited_role_two.role_name}"
+    database_name = "admin"
+  }
+}
+
+```
+
+## Argument Reference
+
+* `project_id` - (Required) The unique ID for the project to create the database user.
+* `role_name` - (Required) Name of the custom role. 
+
+	-> **IMPORTATN** The specified role name can only contain letters, digits, underscores, and dashes. Additionally, you cannot specify a role name which meets any of the following criteria:
+
+	* Is a name already used by an existing custom role in the project
+	* Is a name of any of the built-in roles
+	* Is `atlasAdmin`
+	* Starts with `xgen-`
+
+
+### Actions
+Each object in the actions array represents an individual privilege action granted by the role. It is an required field.
+
+* `action` - (Required) Name of the privilege action. For a complete list of actions available in the Atlas API, see Custom Role Actions.
```

Comment:
Add the link and a bit more detail:
Name of the privilege action. For a complete list of actions available in the Atlas API, see 
[Custom Role Actions] (https://docs.atlas.mongodb.com/reference/api/custom-role-actions).
Note: The privilege actions available to the Custom Roles API resource represent a subset of the privilege actions available in the Atlas Custom Roles UI.